File: src/CSharp/796.RotateString.cs
using System;
using System.Collections;
using System.Collections.Generic;

namespace PeefyLeetCode.RotateString
{
public class Solution {
    public bool RotateString(string A, string B) {
        if (A.Length != B.Length)
            return false;
        return (A + A).Contains(B);
    }
}
}

File: src/Java/701-800/796.RotateString.java


import java.awt.List;
import java.util.ArrayList;
import java.util.Collections;
import java.util.Comparator;
import java.util.LinkedList;
import java.util.Queue;

class Solution {
    public boolean rotateString(String A, String B) {
        if (A.length() != B.length())
            return false;
        return (A + A).contains(B);
    }
}

